packages/help-center/src/components/help-center-search.scss

```diff
@@ -10,7 +10,7 @@
 
 	// Search Bar
 	.card.search-card {
-		border: 1px solid var(--studio-gray-10);
+		border: 1px solid var(--studio-gray-10, #c3c4c7);
```

Comment:
This indicates a wider problem, you can see the buttons are missing their borders too. You need to `@import "@automattic/calypso-color-schemes";` in this file.

Reply:
We don't want to load calypso-color-schemea on v2 so I guess this is a quick way to fix the issue unless there are still more 🙈

Reply:
Yeah there will be many. We faced this issue a couple of times. Maybe make your own color scheme with a few vars? You can browse the Help Center, and you'll see missing borders in a few places.

Reply:
Load styles by the `<HelpCenterApp />` by https://github.com/Automattic/wp-calypso/pull/103761/commits/bb268d63724dc13bf474ad12b0efe7235ca34b8b. I'm not sure but we may need to get rid of the calypso-color-schemes and replace it with core styles.

Reply:
Referring to https://github.com/Automattic/wp-calypso/pull/103761#issuecomment-2914932655, I don't think we're bundling them within client/dashboard. Instead, they're split into `async-load-help-center-app.js`, `async-load-help-center-button.js`, `async-load-help-center-app.css

Referring to https://github.com/Automattic/wp-calypso/pull/103761#issuecomment-2914932655, I don't think we're bundling these within `client/dashboard`. Instead, they're being split into separate chunks: `async-load-help-center-app.js`, `async-load-help-center-button.js`, and `async-load-help-center-app.css`. Am I missing anything?

![image](https://github.com/user-attachments/assets/c80b86df-24e1-4f2e-ad06-15f384be1246)

Reply:
Good point, you're right @arthur791004 this are async loaded and loaded from separate bundles. 

That said, it's still not perfect, my reading of the code is that these asynchronous bundles are always loaded because the help center button is always rendered, what I was thinking is that the idea would be for the button to just be part of the dashboard normally but when you click it, the async loading starts. This prevents the help center from impacting the initial loading. 

Maybe that's too complex to achieve, but at least worth thinking about to see how simple/complex could that be?

Reply:
https://github.com/Automattic/wp-calypso/pull/103761/commits/1d076d154cfecb5ac3a5bf3c54bbe658a5f2e0b5 tried to use the store name directly and it seems better than previous one 🙂

Reply:
To be honest, I don't know :). I'm kind of afraid a bit about the precedent of loading a "@wordpress/data" store in the dashboard but I also don't have alternatives.

Also I'm not familiar enough with how data stores are used in calypso/help center, how they get initialized, do we use them as globals, are we sure we're using the right store or are we using a different store for each "chunk"... Questions I can't answer myself. 

But I'm happy with the progress so far and happy to move forward.